scipy/spatial/src/distance_impl.h

```diff
@@ -309,6 +309,38 @@ jaccard_distance_char(const char *u, const char *v, const npy_intp n)
     return (double)num / denom;
 }
 
+
+static NPY_INLINE double
+jensenshannon_distance_double(const double *p, const double *q, const npy_intp n)
```

Comment:
Isn't this the *divergence* at the moment? Perhaps best to switch over to *distance* here as well?